updateHTML.rb

```diff
@@ -2,39 +2,40 @@
 
 require 'nokogiri'
 require 'open-uri'
-$total_count = 0
```

Comment:
don't use globals unless you really must do so. In this case, you do not need to.

Reply:
Sure, this I will make as an instance variable instead of global variables. Since this was just a temporary solution to the problem I didn't consider these things. But I will do the changes now. Thanks